File: 03.business-os/_archive/commonos_legacy_skeleton_20260425_184835/app_local/PocketSecretary/iphone/PocketSecretary/CommonOSEntry/CommonOSEntryRegistry.swift
import Foundation

enum CommonOSEntryRegistry {
    static func createDefault() -> CommonOSEntryDescriptor {
        CommonOSEntryDescriptor(
            appName: "PocketSecretary",
            commonComponentUsage: [
                "App Shell",
                "Card",
                "List",
                "Search Bar",
                "Filter Panel",
                "Status Chip",
                "Toast",
                "Offline Queue Status UI",
                "Sync Retry UI",
                "Conflict Review UI"
            ],
            variantUsage: [
                "panel.sync",
                "panel.conflict",
                "input.compact",
                "button.primary",
                "card.standard"
            ]
        )
    }
}

File: 03.business-os/PocketSecretary/android/app/src/main/java/com/lsam/PocketSecretary/runtime/commonos/entry/CommonOSEntryRegistry.java
package com.lsam.PocketSecretary.runtime.commonos.entry;

import java.util.ArrayList;
import java.util.List;

public final class CommonOSEntryRegistry {

    public CommonOSEntryDescriptor createDefault() {
        final List<String> components = new ArrayList<>();
        components.add("App Shell");
        components.add("Card");
        components.add("List");
        components.add("Search Bar");
        components.add("Filter Panel");
        components.add("Status Chip");
        components.add("Toast");
        components.add("Offline Queue Status UI");
        components.add("Sync Retry UI");
        components.add("Conflict Review UI");

        final List<String> variants = new ArrayList<>();
        variants.add("panel.sync");
        variants.add("panel.conflict");
        variants.add("input.compact");
        variants.add("button.primary");
        variants.add("card.standard");

        return new CommonOSEntryDescriptor("PocketSecretary", components, variants);
    }
}
